articles/sorting-algorithms-in-js/index.md

```diff
@@ -0,0 +1,510 @@
+Sorting can be referred to as an operation performed to arrange records in some particular order. The arrangement is performed based on the value of each record present. The order applied can either be ascending or descending. Sorting algorithms are instructions given to a computer to arrange elements in a particular order.
+
+### Prerequisites
+
+- You should have some basic knowledge of JavaScript.
+
+- You should have [Node.js](#https://nodejs.org/en/) installed on your computer.
+
+### Overview
+
+- [Categories of sorting algorithms](#categories-of-sorting-algorithms)
+
+- [Efficiency of sorting algorithms](#efficiency-of-sorting-algorithms)
+
+- [Sorting algorithms](#sorting-algorithms)
```

Comment:
In the Overview section, please add the list of sorting algorithms you are discussing in the article.

Reply:
fixed.